src/index.js

```diff
@@ -1,7 +1,13 @@
 import Phaser from 'phaser';
+import LobbyScene from './scenes/LobbyScene';
 import SampleScene from './scenes/SampleScene';
 import SampleShipScene from './scenes/SampleShipScene';
-import EventBus from './helpers/EventBus';
+import  EventBus from './helpers/EventBus';
+import {EVENTS} from './helpers/enums';
+
+// Hacky way to load all pngs for phaser to use.
+function requireAll(r) { r.keys().forEach(r); }
+requireAll(require.context('./assets', true, /\.png$/));
```

Comment:
Is this for webpack+phaser?

Reply:
yep. Since phaser tries to get the png sprite page through a http call not through a require webpack does not know to load those resourcs